tests/support_server/windows/win2019_config.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install VDMP tools to enable network drivers correctly.
+#
+# Maintainer: mmartins <mmartins@suse.com>
+
+#use base 'basetest' ;
```

Comment:
```suggestion
```

Reply:
1. Há! It was escapa for me. Fixed. Thanks